ember/app/ui/components/identity-memberships/template.hbs

```diff
@@ -5,18 +5,37 @@
 >
   <h2>{{t "component.identity-memberships.title"}}</h2>
 
-  {{#if this.fetchInterests.isRunning}}
+  {{#if this.fetchMemberships.isRunning}}
     <LoadingSpinner />
   {{else}}
     {{#if this.memberships.length}}
       <ul class="uk-list uk-list-divider">
         {{#each this.memberships as |membership|}}
           <li class="uk-flex uk-flex-between uk-flex-middle">
             <div>
-              <strong>{{membership.organisation.organisationName}}</strong><br>
+              {{#if membership.isInactive}}
+                <del>
```

Comment:
Is the `del` here just to highlight that the membership is inactive?

Reply:
Exactly